### ISSUE TITLE
Normalize Admin1 location names to match geographic areas

### ISSUE THREAD

**Issue by flaneuse** [outbreak-info/outbreak.info#236]
1. The `NAME` field in `features.properties` of the [GADM](https://gadm.org/data.html) geojson (below) needs to match the names provided by GISAID

https://github.com/outbreak-info/outbreak.info/blob/division/web/src/assets/geo/gadm_adm1_simplified.json 

2. Clean up the `division` for sequences where `division == country` by removing the division data, e.g. 

```
country: Australia,
division: Australia
```
--> 

```
country: Australia,
division: null
```

**Comment by flaneuse** [comment]
As a first pass for the MVP @AlaaALatif, we might just want to only include the countries where the names already match and no more work is done.  If you don't have time to normalize -- either by changing the GADM base (probably easier, more hard-code-y) or the GISAID data -- can you give me a list of good countries and we'll hide the ones w/ mismatches till later?

**Comment by flaneuse** [comment]
@gkarthik @AlaaALatif @juliamullen just realized: because the location names are lowercase normalized, they're not going to match to GADM for things like "Centre-Val de Loire".  To make them "pretty", I'm running a titleCase function on the names returned from ES to turn it into the awful "Centre-val De Loire".

-- strongly thinking in v.2 we should save a lowercased searchable name and a pretty formatted name which matches the GADM shapefile.  The endpoints would return `name` for all functions except the country/state lookups, which would call `name_searchable` (or something).

**Comment by flaneuse** [comment]
And/or match based on a location ID, but that'd require a backend lookup to grab the formatted place name too.

**Comment by AlaaALatif** [comment]
Location normalization should now capture all U.S. state-level information submitted to GISAID, as well as admin1-level information from several countries including Canada, Mexico, Brazil, UK, Germany, Spain, Oman, and others. We can always add more normalization to capture this info for remaining countries, but this should good coverage as a starting point.

An updated notebook with location normalization can be found [here](https://github.com/andersen-lab/bjorn/blob/master/notebooks/gisaid_feed_processing.ipynb)

Going to close this after complete deployment and sanity checks.

**Comment by flaneuse** [comment]
Closed with #272
